python/mxnet/ndarray/contrib.py

```diff
@@ -363,3 +362,97 @@ def _func_wrapper(loop_vars):
                 ["  Step %d, shape is %s" % (i, str(x.shape)) for i, x in enumerate(items)]
             ))
     return stacked_outputs, list(loop_vars)
+
+def cond(cond_func, then_func, else_func, inputs):  # pylint: disable=redefined-outer-name
```

Comment:
can we remove inputs and have tf style interface?

Reply:
Good idea. Just finished the API change. Thanks!